### PR TITLE
Make interventricular sulcus exact synonym of interventricular groove

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/uberon/issues/2703

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -102919,7 +102919,7 @@ id: UBERON:0005455
 name: interventricular groove
 def: "Either of the anterior and posterior grooves on the surface of the heart that lie over the interventricular septum and join at the apex." [http://www.merriam-webster.com/medical/interventricular%20sulcus]
 subset: pheno_slim
-synonym: "interventricular sulcus" RELATED [FMA:7223]
+synonym: "interventricular sulcus" EXACT [FMA:7223]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:102922]: References to support this being a synonym:
https://www.merriam-webster.com/medical/interventricular%20sulcus
https://medical-dictionary.thefreedictionary.com/interventricular+groove

**Review by bvarner-ebi — APPROVED** [review]
Suggestion: consider adding references to support the synonym. See inline comment.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:102922]:
```suggestion
synonym: "interventricular sulcus" EXACT [FMA:7223, https://www.merriam-webster.com/medical/interventricular%20sulcus]
```

**Comment by bvarner-ebi** [comment]
> Suggestion: consider adding references to support the synonym. See inline comment.

I now see the synonym is indicated here: https://bioportal.bioontology.org/ontologies/FMA/?p=classes&conceptid=http%3A%2F%2Fpurl.org%2Fsig%2Font%2Ffma%2Ffma7223&jump_to_nav=true.
Additional references are not necessary.

**Pull request merged**